File: ios/EscalaMinisterial/Models/Indisponibilidade.swift
import Foundation

struct Indisponibilidade: Codable, Identifiable {
    var id: Int?
    var ministroId: Int?
    var data: String
    var horarioInicio: String?
    var horarioFim: String?
    var motivo: String?

    var dataFormatada: String {
        guard let date = DateUtils.parseDate(data) else { return data }
        return DateUtils.formatDate(date)
    }
}

File: android/feature/ministros/src/main/java/com/escala/ministerial/feature/ministros/domain/model/Indisponibilidade.kt
package com.escala.ministerial.feature.ministros.domain.model

import java.time.LocalDate

data class Indisponibilidade(
    val id: Long,
    val ministroId: Long,
    val data: LocalDate,
    val horarioInicio: String?,
    val horarioFim: String?,
    val motivo: String?,
)
